components/lib/crash/src/test/java/mozilla/components/lib/crash/CrashReporterTest.kt

```diff
@@ -68,11 +71,13 @@ class CrashReporterTest {
 
     @Test
     fun `CrashReporter will submit report immediately if setup with Prompt-NEVER`() {
+        val scope = TestCoroutineScope(testDispatcher)
```

Comment:
We could have put this in the `setup` but it's okay. 😛

Reply:
Sure.  That's a good idea.  Not in `setup` but I'll make it a property.